package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@freestar/cordova-plugin-freestar",
-  "version": "1.1.2",
+  "version": "1.1.3",
```

Comment:
@kkawai in the future there is no need to manually bump this version. its handled during publish from master branch